README.md

```diff
@@ -19,6 +19,12 @@ ember-cli-eslint
 Installation
 ------------------------------------------------------------------------------
 
+ESLint 4 (for Node 4 and above):
+
+```
+ember install ember-cli-eslint@4
+```
+
 ESLint 3 (for Node 4 and above):
```

Comment:
Maybe just give `ember install ember-cli-eslint` as instructions and add that if a specific ESLint version is desired the corresponding ember-cli-eslint version has to be used?